CHANGELOG.md

```diff
@@ -1,23 +1,33 @@
-## 2018-01-05 - Supported Release 1.4.1
+# Changelog
 
-The DSC module received a few bug fixes related to the PowerShell handler and a security fix in the form of enabling users to specify credentials as [Sensitive](https://puppet.com/docs/puppet/latest/lang_data_sensitive.html).
+All notable changes to this project will be documented in this file.
+ 	 
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## Fixed
+## [Unreleased]
+
+## [1.4.1] - 2018-01-05 - Supported Release 1.4.1
```

Comment:
Nit: There's no need to specify supported release or the number a second time. Metadata indicates support, first part of the header indicates version.

Reply:
I was following https://github.com/puppetlabs/puppetlabs-reboot/blob/master/CHANGELOG.md#110---2015-07-28---supported-release-110 as an example, but agree that its redundant and repetitive.